### PR TITLE
Implement e2e test for Stackdriver event exporter

### PR DESCRIPTION
**Pull request by crassirostris**
Fixes https://github.com/kubernetes/kubernetes/issues/47963

/cc @piosz

### CODE REVIEW

File: test/e2e/cluster-logging/sd_events.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```
Comment (x13n) [inline test/e2e/cluster-logging/sd_events.go]: 2017?

**Comment by x13n** [comment]
Otherwise, lgtm

File: test/e2e/cluster-logging/sd_events.go (hunk 1 above)
Comment (crassirostris) [inline test/e2e/cluster-logging/sd_events.go]: Ack, copy-paste

File: test/e2e/cluster-logging/sd_events.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+package e2e
+
+import (
+	"time"
+
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	"k8s.io/apimachinery/pkg/util/wait"
```
Comment (crassirostris) [inline test/e2e/cluster-logging/sd_events.go]: Ack, Idea's auto-complete

**Comment by crassirostris** [comment]
@x13n Done, PTAL

**Comment by x13n** [comment]
Let's see if casting this spell will work:

**Comment by x13n** [comment]
Ok, one more spell.

**Comment by piosz** [comment]
@caesarxuchao @dchen1107 can we cherry-pick this?

**Comment by caesarxuchao** [comment]
The change is non-trivial. I'd prefer punt to 1.7.1. Is there any reason this need to get in 1.7.0?

**Comment by crassirostris** [comment]
@caesarxuchao To verify that the new functionality in 1.7 is working, after all patches. This is only a missing e2e test.

**Comment by caesarxuchao** [comment]
Does `test/e2e/cluster-logging/sd_utils.go` only affect your e2e tests?

**Comment by crassirostris** [comment]
@caesarxuchao Yes, it only affects Stackdriver logging and now events tests.

**Comment by caesarxuchao** [comment]
Ok. I'll cherrypick. Note that the test won't have soak time because we are planning to cut the release tomorrow.

**Comment by crassirostris** [comment]
@caesarxuchao OK, thanks!

**Comment by piosz** [comment]
@caesarxuchao I'm fine with not blocking the release if the test will fail.

**Pull request merged**